pkg/config/tlscfg/options_unix.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2021 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !windows
+
+package tlscfg
+
+import "crypto/x509"
+
+func appendCerts(rootCAs *x509.CertPool) (*x509.CertPool, error) {
```

Comment:
I mostly copied the code from sensu-go PR but now reading about it it seems like the goal of `appendCerts` is to load all certificates that are installed in the machine.
On unix systems, SystemCertPool (https://golang.org/pkg/crypto/x509/#SystemCertPool) already does that so we are just defining the function without doing anything. 
On windows, since SystemCertPool doesn't load those installed certificates, we are doing it ourselves.